appserver/web/war-util/src/main/java/org/glassfish/web/loader/WebappClassLoader.java

```diff
@@ -64,20 +64,15 @@
 import com.sun.appserv.server.util.PreprocessorUtil;
 import com.sun.enterprise.deployment.Application;
 import com.sun.enterprise.deployment.util.DOLUtils;
+import com.sun.enterprise.security.integration.DDPermissionsLoader;
+import com.sun.enterprise.security.integration.PermsHolder;
 import com.sun.enterprise.util.io.FileUtils;
 import org.apache.naming.JndiPermission;
-import org.apache.naming.resources.DirContextURLStreamHandler;
-import org.apache.naming.resources.JarFileResourcesProvider;
-import org.apache.naming.resources.ProxyDirContext;
-import org.apache.naming.resources.WebDirContext;
-import org.apache.naming.resources.Resource;
-import org.apache.naming.resources.ResourceAttributes;
+import org.apache.naming.resources.*;
```

Comment:
I would stay with a list of classes if it is less than 11.

Reply:
That's up for debate - personally I hate using `*` unless I'm literally using every single class :P

Reply:
Yes, my experience is that if you use more than 10 classes from the same package in imports, usually the new class asks to be refactored, so finally we end up in the same nice state :)